z2/resources/chain-specs/zq2-prototestnet.toml

```diff
@@ -39,4 +39,5 @@ consensus.forks = [
     { at_height = 12693600, scilla_call_respects_evm_state_changes = true },
     { at_height = 12884400, only_mutated_accounts_update_state = true, scilla_block_number_returns_current_block = true },
     { at_height = 12931200, scilla_maps_are_encoded_correctly = true },
+    { at_height = 14857110, apply_state_changes_only_if_transaction_succeeds = true, apply_scilla_delta_when_evm_succeeded = true},
```

Comment:
Sorry, I forgot to mention that this needs to be set in `get_forks` of `chain.rs` too, otherwise the GCP nodes won't use the configuration.